### PR TITLE
8697 fixes from 8672 to qat

### PR DESCRIPTION
**Pull request by brianpetway**
https://federal-spending-transparency.atlassian.net/browse/DA-8697

### CODE REVIEW

**Comment by anjenkin** [comment]
The heights are removed in this branch.   If this is from the cherry pick maybe we need to just re-add the fix for that commit to the qat branch by just updated the code instead of by using the commit

**Comment by brianpetway** [comment]
added the heights for the svgs. ready to review again.

File: src/page-sections/federal-covid-funding/tracking/index.jsx [hunk 1]
```diff
@@ -750,7 +749,7 @@ export default function Tracking(props) {
 								className={phaseDetail['govtTotal'].className}
 								src={SectionTag.svg}
 								width={SectionTag.width}
-								height={SectionTag.height}
```
Comment (anjenkin) [inline src/page-sections/federal-covid-funding/tracking/index.jsx:753]: this height line needs to be added back

**Review by anjenkin — CHANGES_REQUESTED** [review]
see one my change needed for the image heights inline

**Comment by brianpetway** [comment]
Added that height prop.

**Review by anjenkin — APPROVED** [review]

**Pull request merged**